src/js/components/DeploymentComponent.jsx

```diff
@@ -16,10 +19,75 @@ var DeploymentComponent = React.createClass({
 
   getInitialState: function () {
     return {
-      loading: false
+      loading: false,
+      continueButtonsLoadingState: {}
     };
   },
 
+  componentWillMount: function () {
+    DeploymentStore.on(DeploymentEvents.CONTINUE_MIGRATION_SUCCESS,
+      this.onContinueMigrationSuccess);
+    DeploymentStore.on(DeploymentEvents.CONTINUE_MIGRATION_ERROR,
+      this.onContinueMigrationError);
+  },
+
+  componentWillUnmount: function () {
+    DeploymentStore.removeListener(DeploymentEvents.CONTINUE_MIGRATION_SUCCESS,
+      this.onContinueMigrationSuccess);
+    DeploymentStore.removeListener(DeploymentEvents.CONTINUE_MIGRATION_ERROR,
+      this.onContinueMigrationError);
+  },
+
+  getContinueButton: function (action) {
+    if (action.isWaitingForUserDecision === true) {
```

Comment:
I would prefer the inversed the case.
A function returning null as default is unexpected to me.
```JavaScript
if (!action.isWaitingForUserDecision) {
  return null;
}
```